render/detailed/connections.go

```diff
@@ -0,0 +1,197 @@
+package detailed
```

Comment:
This file may be easier to read if you flip the order so the primary functions (`makeIncomingConnectionsTable` and `makeOutgoingConnectionsTable`) were first, then the helpers afterwards.